sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatClientBuilder.java

```diff
@@ -166,9 +168,10 @@ public ChatAsyncClient buildAsyncClient() {
         } else {
             Objects.requireNonNull(communicationUserCredential);
             Objects.requireNonNull(httpClient);
+            CommunicationTokenCredential tokenCredential = new CommunicationTokenCredential(communicationUserCredential);
 
             pipeline = createHttpPipeline(httpClient,
-                new ChatUserCredentialPolicy(communicationUserCredential),
```

Comment:
Looks like you should also remove the ChatUserCredentialPolicy file as it is not used anymore

Reply:
Just removed it!

Reply:
I see you already took care of it